AppCenterIdentity/AppCenterIdentity/Internals/MSIdentityPrivate.h

```diff
@@ -32,9 +33,9 @@ static NSString *const kMSIdentityMSALAccountHomeAccountKey = @"MSIdentityMSALAc
 @property(nonatomic, nullable) MSIdentityConfig *identityConfig;
 
 /**
- * The flag that indicates a user requested signIn before it is configured.
+ * Ingestion instance (should not be deallocated).
```

Comment:
Any reason for this one? Why shouldn't it be deallocated?

Reply:
This is the reason for the PR. From bug desc (my initial investigation):

> 1. Disable internet connection (wi-fi or airplane mode etc)
> 1. Start the application or re-enable Identity module
> 1. Enable internet connection
> 1. Try to sign in. 
> 1. No UI. No error logs.
> 
> The problem is that `MSIdentityConfigIngestion` deallocated and retries logic doesn't work.